app/src/main/AndroidManifest.xml

```diff
@@ -52,7 +52,8 @@
             android:name=".content.task.StarActivity"
             android:screenOrientation="fullSensor" />
         <activity android:name=".content.task.LessonCompleteActivity" />
-        <activity android:name=".authentication.StudentImageCollectionActivity" />
+        <activity android:name=".authentication.StudentImageCollectionActivity"
+            android:screenOrientation="landscape"/>
```

Comment:
@jogrimst  with the change in the manifest it should switch to landscape  mode (like with some other apps on the tablet)